src/commands/helm-conftest.yaml

```diff
@@ -0,0 +1,13 @@
+---
+parameters:
+  chart:
+    type: "string"
+description: Test conftest policies
+steps:
+  - run: |
+      if [ -d "helm/<<parameters.chart>>/ci" ]; then
+        for t in $(ls helm/<<parameters.chart>>/ci/*.yaml); do helm template --values $t helm/<<parameters.chart>> | conftest test --update https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.16-deprek8ion.rego,https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.17-deprek8ion.rego,https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.18-deprek8ion.rego,https://raw.githubusercontent.com/swade1987/deprek8ion/master/1.19-deprek8ion.rego azure-operator -; done
+      else
```

Comment:
Do we want this else? I think we always want to check the thing in the `else` block.

Reply:
If there are values files in the `ci` folder, running just `helm template` will fail because of potential missing values (thinkin `installations` values).
So we want to run `conftest` without passing any values files to `helm template`, or run `conftest` for every potential values file, to test all potential code paths.

I don't know how to do it without the `if/else`